.pre-commit-config.yaml

```diff
@@ -23,6 +23,7 @@ repos:
         third_party/SuiteSparse/AMD/.*|
         third_party/identify_stream_usage/.*|
         examples/external-lib-interfacing/external-lib-interfacing.cpp
+        core/base/workspace_aliases.hpp
```

Comment:
why is the header excluded here? I thought it would not change the copyright from previous years.